### PR TITLE
Notification route

### PR DESCRIPTION
**Pull request by BALAJI141031**
files to check:
1.routes > notification(index.js,index.css)

### CODE REVIEW

File: src/backend/controllers/PostController.js [hunk 1]
```diff
@@ -63,8 +64,48 @@ export const getAllUserPostsHandler = function (schema, request) {
...
+//   const user = requiresAuth.call(this, request);
+//   try {
+//     if (!user) {
+//       return new Response(
+//         404,
+//         {},
+//         {
+//           errors: [
+//             "The username you entered is not Registered. Not Found error",
+//           ],
```
Comment (mayuresh-tech) [inline src/backend/controllers/PostController.js:77]: You can remove commented code

**Review by mayuresh-tech — APPROVED** [review]
Good to merge!

File: src/routes/NotificationRoute/index.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            <div className="icon-div">
+              {index % 2 !== 0 ? (
+                <BsHeartFill className="notification-icon like-notification" />
+              ) : (
+                <FaCommentAlt className="notification-icon comment-notification" />
+              )}
+            </div>
+            <div>
+              <div className="notification-from">
+                <img
```
Comment (anshulraheja) [inline src/routes/NotificationRoute/index.js:34]: try to give 'alt' attribute for img tag. Rest of the code is good to merge

**Review by anshulraheja — APPROVED** [review]

File: src/components/AnalyticsCta/index.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    margin-left:5px;
+}
+
+.analytics-div > div{
+    margin-right:20px
+}
+
+.style-analytics-icon{
+    
+    fill: #F73D93;
```
Comment (deekshithmd) [inline src/components/AnalyticsCta/index.css:16]: color variable recommended

File: src/components/AnalyticsCta/index.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+export default function AnalyticsIcon({ icon, post }) {
+  const { _id: postid, likes, bookMarked } = post;
+  const [localBookMark, setBookMark] = useState(false);
+  console.log(bookMarked, "i want to know status", localBookMark);
+  const [liked, toggleLike] = useState(false);
+
+  // like handler working alone but in seires with bookmarks giving serilization error
+  const toggleLikeHandler = async () => {
+    toggleLike((likeStatus) => !likeStatus);
+    // try {
```
Comment (deekshithmd) [inline src/components/AnalyticsCta/index.js:21]: Remove these commented code if not required

File: src/components/CreateTweet/index.css [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    top: 0;
+    right: 0;
+    left: 0;
+    bottom: 0;
+    background-color: rgba(0,0.1,0.1,0.2);
+    z-index: 99; 
+}
+
+.cancel-cta-div{
+    /* border-bottom: 1px solid; */
```
Comment (deekshithmd) [inline src/components/CreateTweet/index.css:17]: remove commented code

File: src/components/CreateTweet/index.js [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+
+  const hiddenFileInput = useRef(null);
+
+  useEffect(() => {
+    if (fromEdit.editStatus) {
+      (async () => {
+        try {
+          const tweetResponse = await getTweet(fromEdit.tweetId);
+
+          console.log("during updation", tweet);
```
Comment (deekshithmd) [inline src/components/CreateTweet/index.js:49]: remove console.log

File: src/components/CreateTweet/index.js [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+          setTweet({
+            type: REDUCER_CONSTANTS.CAPTION,
+            payload: tweetResponse.caption,
+          });
+          setTweet({
+            type: REDUCER_CONSTANTS.TWEETTEXT,
+            payload: tweetResponse.content,
+          });
+
+          console.log("yes i'm getting my response back", tweetResponse);
```
Comment (deekshithmd) [inline src/components/CreateTweet/index.js:59]: same here...console.log

File: src/components/CreateTweet/index.js [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+              <div className="icons-section">
+                <BiUpload
+                  className="crate-tweet-icon"
+                  onClick={() => {
+                    hiddenFileInput.current.click();
+                  }}
+                />
+                <div className="emojis-section">
+                  <BsEmojiSmile
+                    className="crate-tweet-icon"
```
Comment (deekshithmd) [inline src/components/CreateTweet/index.js:127]: Try to find all commented codes and remove if not required

File: src/components/Cta/index.css [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+.secondary-cta{
+    background-color: palegreen;
+}
+
+#cta{
+    text-align: center;
+    width: 100%;
+    border-style: none;
+    padding: 0.3em;
```
Comment (deekshithmd) [inline src/components/Cta/index.css:16]: using px will be good for values less than 1 rem

**Review by deekshithmd — APPROVED** [review]

**Pull request merged**